src/main/io/osd.c

```diff
@@ -922,7 +922,7 @@ void osdUpdate(timeUs_t currentTimeUs)
         // redraw values in buffer
         osdRefresh(currentTimeUs);
     } else {
-        // rest of time redraw screen 10 chars per idle so it doesn't lock the main idle
+        // rest of time redraw screen
```

Comment:
I would change the comment to `// rest of time redraw screen partially to avoid long pauses`.